pkg/k8s/cnp_test.go

```diff
@@ -451,21 +451,16 @@ func benchmarkCNPNodeStatusController(integrationTest bool, nNodes int, nParalle
 	}()
 
 	var cnpStore cache.Store
-	switch {
-	case k8sversion.Capabilities().UpdateStatus:
-		// k8s >= 1.13 does not require a store
```

Comment:
This is also relevant to point out given my above question. It's hard to tell what versions matter for `UpdateStatus` and what we as Cilium should do as a result.

Reply:
See previous comment